pkgs/development/python-modules/huepy/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  buildPythonPackage,
+  pytestCheckHook,
+  setuptools,
+  pytest,
+}:
+
+buildPythonPackage rec {
+  pname = "huepy";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "huepy";
+    inherit version;
+    hash = "sha256-Wym+73lzEvt2BhiLxc2Y94q49+AVdkJ6kxLxybILdZ0=";
+  };
+
+  build-system = [ setuptools ];
+
+  buildInputs = [ pytest ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "huepy" ];
+
+  doCheck = false;
+
+  meta = {
+    description = "Print awesomely in terminals";
+    homepage = "https://pypi.org/project/huepy/";
+    license = "licenses.gpl3";
```

```suggestion
    license = "licenses.gpl3Plus";
```
please use `gpl3Only` or `gpl3Plus` instead of `gpl3`.